docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md

```diff
@@ -0,0 +1,399 @@
+linkTitle: "Create Automanage Machine Configuration custom configurations "
+weight: 8
+description: >
+---
+
+## Create Automanage Machine Configuration custom configurations for Linux
+
+The following Jumpstart scenario will guide you on how to create and assign Automanage Machine Configuration custom configurations to an Azure Arc-enabled Linux server. Automanage makes it easy to follow best practices in reliability, security, and management for Azure Arc-enabled servers using Azure services such as [Azure Update Management](https://docs.microsoft.com/azure/automation/update-management/overview) and [Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/vm/vminsights-overview).
+
+While the use of custom configurations in Automanage Machine Configuration feature is based on PowerShell Desired State Configuration (DSC), there are [Changes to behavior in PowerShell DSC for Machine Configuration](https://learn.microsoft.com/en-us/azure/governance/machine-configuration/machine-configuration-custom?view=dsc-2.0) to be aware of, the most significant being the use of PowerShell 7.
+
+By the end of this scenario, you will have both Linux and Windows Azure Arc-enabled servers with Automanage Machine Configuration custom configurations assigned.
+
+This scenario starts at the point where you already deployed **[Jumpstart ArcBox for IT Pros](https://azurearcjumpstart.io/azure_jumpstart_arcbox/itpro/)** and have 5 Azure Arc-enabled servers in the resource group is deployed to visible as resources in Azure.
+
+![Screenshot of Azure Portal showing Azure Arc-enabled servers](./01.png)
+
+The custom configurations are written using PowerShell Desired State Configuration (DSC), and needs to be authored from a machine running the target operating system for the configurations (Linux-machine for Linux configurations and Windows-machine for Windows configurations).
+
+After the configurations has been authored and published into your Azure environment, they can be assigned to any Linux-based Azure Arc-enabled server (or Azure VM) in your environment.
+
+This scenario will assign it to the resource group ArcBox is deployed to.
+
+## Base requirements - configuration authoring
+
+Operating system:
+
+- Ubuntu 18 (required by the GuestConfiguration module)
+
+In this scenario, we will be using the ArcBox Client virtual machine for the configuration authoring - and connect to a nested Linux VM.
+
+You can [connect to the ArcBox machine as described in the documentation]( https://azurearcjumpstart.io/azure_jumpstart_arcbox/itpro/#connecting-to-the-arcbox-client-virtual-machine) and perform the following:
+
+- Open Visual Studio Code from the desktop shortcut.
+- Install the [Remote SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh).
+
+![Screenshot of Remote SSH extension](./14.png)
+
+Open Hyper-V Manager and determine the IP address of the ArcBox-Ubuntu-01 VM:
+
+![Screenshot of Remote SSH extension](./15.png)
```

Comment:
Most of the images have the same title. Please, use a descriptive title per each image and start each with "Screenshot of.."

Reply:
Resolved by https://github.com/microsoft/azure_arc/pull/1767/commits/8953a05390b4934d461aa0ae0a50f5c1dbe96887